cpp/third_party/mongoose/src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+project(Mongoose)
+
+add_library(mongoose mongoose.c)
```

Comment:
I think you only need `mongoose.c` and `mongoose.h` per the docs - I think the rest is just for their project internal code organization, so not sure we need to check it in

Reply:
Here's what I mean, I think you should only need this: https://github.com/benjischwartz/CellSim/pull/5